docs/content/en/docs/getting-started/_index.md

```diff
@@ -3,9 +3,9 @@ title: Installation
 linkTitle: Installation
 aliases:
     /docs/getting-started/production-environment/
+    /docs/getting-started
 main_menu: true
 weight: 20
-no_list: true
```

Comment:
why are we removing this? (I'm not familiar with what this does)

Reply:
So this page currently doesn't show the subsections https://anywhere.eks.amazonaws.com/docs/getting-started/
while all the other ones do, for ex this one https://anywhere.eks.amazonaws.com/docs/packages/

Removing this just makes it so that the installation page shows all the subsections kinda like this 
![image](https://github.com/aws/eks-anywhere/assets/12872943/7fcd5693-e2e3-4f60-b25b-fa87302597c2)

Reply:
Yea much better to have these listed these out